core/modules/views/views.module

```diff
@@ -2188,3 +2188,30 @@ function views_autoload_info() {
     'ViewsSpecialBlock' => 'includes/views.special_block.inc',
   );
 }
+
+/**
+ * Cleans a CSS identifier, taking into account views tokenization.
+ *
+ * @param sring &$class
+ *   A single CSS class, as entered into the Views user interface.
+ * @param object $plugin
+ *   The complete view object for tokenization.
+ * @param string $row_index
+ *   The index indicating the row of the view.
+ *
+ * @return string $class
+ *   A sanitized clean css identifier with tokens replaced.
+ */
+function _views_tokenized_clean_css_identifier(&$class, $plugin, $row_index) {
```

Comment:
`$class` should be returned as a string, rather than modified by reference. That matches the behavior of all other string manipulation functions (e.g. str_replace(), strip_tags(), backdrop_clean_css_identifier(), etc.)

Reply:
Fixed.